packages/nsq/nsq.0.4.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Ryan Slade <ryanslade@gmail.com>"
+authors: "Ryan Slade <ryanslade@gmail.com>"
+homepage: "https://github.com/ryanslade/nsq-ocaml"
+bug-reports: "https://github.com/ryanslade/nsq-ocaml/issues"
+dev-repo: "git+https://github.com/ryanslade/nsq-ocaml.git"
+build: [["jbuilder" "build" "-p" name "-j" jobs]]
```

Comment:
be aware that jbuilder syntax is currently being deprecated. See https://dune.build/blog/second-step-deprecation/